pkgs/development/python-modules/backports_shutil_which/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, fetchPypi, buildPythonPackage }:
+
+buildPythonPackage rec {
+  pname = "backports.shutil_which";
+  version = "3.5.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "16sa3adkf71862cb9pk747pw80a2f1v5m915ijb4fgj309xrlhyx";
+  };
+
+  # Tests fail: "ValueError: underlying buffer has been detached"
+  doCheck = false;
+
+  meta = with stdenv.lib; {
+    description = "Backport of shutil.which from Python 3.3";
```

Comment:
Maybe python 3 should be disabled here?

Reply:
should be fine: https://github.com/minrk/backports.shutil_which/blob/master/backports/shutil_which.py#L75